gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -0,0 +1,132 @@
+            # Add annotations used in constraint calculations.
+            for data_type in DATA_TYPES:
+                if data_type != "context":
+                    ht = get_sites_resource(data_type, version).ht()
+                else:
+                    ht = context_ht
+
+                # Filtering the Table to chr20, chrX, and chrY for testing if applicable
+                if test:
+                    rg = get_reference_genome(context_ht.locus)
+                    contigs_keep = [
+                        hl.parse_locus_interval(c, reference_genome=rg)
+                        for c in [rg.contigs[19], rg.x_contigs[0], rg.y_contigs[0]]
+                    ]
+                    logger.info(
+                        "Filtering the %s HT to chr20, chrX, and chrY for testing...",
+                        data_type,
+                    )
+                    ht = hl.filter_intervals(ht, contigs_keep)
+
+                # Add annotations from VEP context Table to genome and exome Tables.
+                ht = (
+                    add_vep_context_annotations(ht, context_ht)
+                    if data_type != "context"
+                    else ht
+                )
+
+                # Filter input Table and add annotations used in constraint calculations.
+                full_ht = prepare_ht_for_constraint_calculations(ht)
+                # Filter to locus that is on an autosome or in a pseudoautosomal region.
+                ht = full_ht.filter(full_ht.locus.in_autosome_or_par())
+                ht.write(
+                    get_preprocessed_ht(data_type, version, "autosome_par", test).path,
+                    overwrite=overwrite,
+                )
+
+                # Sex chromosomes are analyzed separately, since they are biologically different from the autosomes.
+                if data_type != "genomes":
+                    filter_x_nonpar(full_ht).write(
+                        get_preprocessed_ht(
+                            data_type, version, "chrx_nonpar", test
+                        ).path,
+                        overwrite=overwrite,
+                    )
+                    filter_y_nonpar(full_ht).write(
+                        get_preprocessed_ht(
+                            data_type, version, "chry_nonpar", test
+                        ).path,
+                        overwrite=overwrite,
+                    )
+            logger.info("Done with preprocessing genome and exome Table.")
+
+    finally:
+        logger.info("Copying log to logging bucket...")
+        hl.copy_log(get_logging_path("constraint_pipeline"))
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--overwrite", help="Whether to overwrite output files", action="store_true"
+    )
+    parser.add_argument(
+        "--version",
+        help=f"Which version of the resource Tables will be used. Default is {CURRENT_VERSION}.",
```

Comment:
```suggestion
        help=(
            "Which version of the resource Tables will be used. Default is"
            f" {CURRENT_VERSION}."
        ),
```

Reply:
Just for curious, were the code here and the other two below formatted by a code style package? If the package does exist, I will add it to this repo as well!